controllers/kubeadmconfig_controller_test.go

```diff
@@ -970,6 +970,67 @@ func TestKubeadmConfigReconciler_ClusterToKubeadmConfigs(t *testing.T) {
 	}
 }
 
+// Exactly one control plane machine initializes if there are multiple control plane machines defined
+func TestKubeadmConfigReconciler_Reconcile_ExactlyOneControlPlaneMachineInitializes(t *testing.T) {
```

Comment:
side topic: i think that `_` in function names fails `golint`
is this file ignored from golint checks?

Reply:
I'm not sure actually..i didn't see anything in the code that indicated tests are skipped. I'd say let's not worry about it for this PR. I've made an issue for it https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm/issues/223